runner/src/main/openapi/runner.yaml

```diff
@@ -674,6 +675,124 @@ components:
           x-field-extra-annotation: "@com.redis.om.spring.annotations.Indexed"
           allOf:
             - $ref: '#/components/schemas/RunnerSecurity'
+      required:
+        - id
+        - name
+        - ownerId
+        - solutionId
+        - runTemplateId
+        - organizationId
+        - workspaceId
+        - creationDate
+        - lastUpdate
+        - ownerName
+        - datasetList
+        - parametersValues
+        - validationStatus
+        - security
+
+    # CRUD Operation Schemas
+    RunnerCreateRequest:
+      type: object
+      description: Request object for creating a new runner
+      properties:
+        name:
+          type: string
+          description: the Runner name
+        description:
+          type: string
+          description: the Runner description
+        tags:
+          type: array
+          description: the list of tags
+          items:
+            type: string
+        solutionId:
+          type: string
+          readOnly: true
+          description: the Solution Id associated with this Runner
+        parentId:
+          type: string
+          description: the Runner parent id
+        runTemplateId:
+          type: string
```

Comment:
As it's marked as required, we can add there some formatting checks:
- minLength (at least to be sure that the name cannot be empty string)